max_x_error: 50 # [pixels]
max_y_error: 50 # [pixels]
max_z_error: 50 # [pixels]
